static/scss/search-page.scss

```diff
@@ -365,3 +365,20 @@
     color: rgba(0,0,0,0.8);
   }
 }
+
+.sk-hierarchical-menu-list {
+  width: 100%;
+
+  .sk-hierarchical-menu-list__root {
+    margin-left: 10px;
+    margin-right: 10px !important;
```

Comment:
these are new classes, we are overriding builtin classes of searchkit. this class already has margin right, here we want suppress that margin